.mvn/gradle-enterprise-custom-user-data.groovy

```diff
@@ -17,7 +17,7 @@ if (System.env.GITHUB_ACTIONS) {
     
    
     def prnumber = System.env.PULL_REQUEST_NUMBER
-    if (prnumber != null) {
+    if (prnumber != null || prnumber.trim() != '') {
```

Comment:
It should be `&&`, right? Not sure about `isBlank()` but I suppose it's also available there?
```suggestion
    if (prnumber != null && !prnumber.isBlank()) {
```

Reply:
Doh. Pushed the wrong commit. Will redo.